help/en/html/apps/DecoderPro/FAQ.shtml

```diff
@@ -1,148 +1,116 @@
-        engine on the track (only one!), clean the wheels and the track and try again.  Error 306 means that JMRI is not getting the expected signal
-        from the command station.  If this happens when you are first setting things up, check your wiring, connections, and com port selection.  
-        This<a href="../Errors.shtml">list of all error messages </a>also provides some suggestions about how to clear them.
+          JMRI will display various errors when it finds something wrong in attempting to
+          communicate with your DCC command station. In particular, Error 301 means that DecoderPro
+          doesn't see a locomotive on the programming track when it is trying to program a decoder.
+          If you do have an engine on the track (only one!), clean the wheels and the track and try
+          again. Error 306 means that JMRI is not getting the expected signal from the command
+          station. If this happens when you are first setting things up, check your wiring,
+          connections, and com port selection. This<a href="../Errors.shtml">list of all error
+          messages</a> also provides some suggestions about how to clear them.
         </dd>
-        
-      
+
         <dt id="lb-xp">Why does my LocoBuffer-II not work in Windows XP?</dt>
 
-        <dd>
-          With Windows XP, your serial port parameters might not
-          be properly set for hardware flow control. To fix this,
-          right-click on My Computer and select Properties. Go to
-          the Hardware tab and click on Device Manager. Open up the
-          section that says Ports (COM &amp; LPT). Right-click on
-          the serial port you have the LocoBuffer hooked to and
-          select Properties. Go to the Port Settings tab and set
-          the options to: <em>19200, 8, None, 1, Hardware</em>.
-          Click <em>OK</em>. Close the Properties panel. Restart
-          JMRI and verify your connection settings.
-        </dd>
-      </dl><!--#include virtual="/Footer.shtml" -->
-    </div><!-- closes #mainContent-->
-  </div><!-- closes #mBody-->
-  <script src="/js/main.js"></script>
-  <!-- FAQ-Tail -->
+        <dd>With Windows XP, your serial port parameters might not be properly set for hardware
+        flow control. To fix this, right-click on My Computer and select Properties. Go to the
+        Hardware tab and click on Device Manager. Open up the section that says Ports (COM & LPT).
+        Right-click on the serial port you have the LocoBuffer hooked to and select Properties. Go
+        to the Port Settings tab and set the options to: <em>19200, 8, None, 1, Hardware</em>.
+        Click <em>OK</em>. Close the Properties panel. Restart JMRI and verify your connection
+        settings.</dd>
+      </dl>
+      <!--#include virtual="/Footer.shtml" -->
+    </div>
+    <!-- closes #mainContent-->
+  </div>
+  <!-- closes #mBody-->
+  <script src="/js/main.js"></script> <!-- FAQ-Tail -->
```

Comment:
This "tidy" construct of putting comments at the end of other lines is confusing.  In this case, the comment at the end of 112 actually applies to the next line 114.  The other example I have found have to do with the "includes" that are put in comments, as in line 5 (which I have noted in your previous PR).  There are some cases where this doesn't happen and it looks like the comment needs an empty line ahead of it to avoid this.   Not a reason to not go ahead with this PR, but I am concerned that important comments (such as "includes") might get lost to the casual looker.